### DISCUSSION TITLE
We were unable to find a Classroom assignment for your account.

### DISCUSSION THREAD

**Discussion by CodeCraftman2** [community/community#120535] [GitHub Education] [answered]
@everyone Hi everyone,

Hi Community,

I'm having trouble accessing my GitHub Classroom assignment despite being enrolled. My repository shows the task completed with screenshots attached. Any advice on how to resolve this?

Thanks,

![Screenshot 2024-04-21 160037](https://github.com/community/community/assets/143512277/c2dcec19-c683-4219-9c01-c618da183267)
![Screenshot 2024-04-21 160100](https://github.com/community/community/assets/143512277/b188b1ee-da61-43c1-97f3-e3ddd315b9f6)
![Screenshot 2024-04-21 160125](https://github.com/community/community/assets/143512277/fef88786-4814-42d5-bd6b-2442809ec366)
![Screenshot 2024-04-21 160145](https://github.com/community/community/assets/143512277/30e24415-b714-4ca6-b623-fa1c7b6eeefd)

**Comment by gurjeetsinghvirdee** [comment]
Hey @CodeCraftman2 

Check the third image it was your assignment repository, follow the steps mentioned in the Readme to complete the task, and the last image, which was not a task but rather a GitHub status page, shows all APIs working correctly.

**Reply by CodeCraftman2** [reply]
Thanks for the clarification! I've completed the tasks as instructed and attached proof in the images. Despite this, I'm still encountering the issue. Any further suggestions?

Appreciate your help!
![image_2024-04-21_173156326](https://github.com/community/community/assets/143512277/2d8f4c01-bd9a-46da-bd27-6617f4c455fb)

**Reply by gurjeetsinghvirdee** [reply]
I think we can't do anything about this because this task was added back in 2023 for a limited time. Since it started in September 2023, I believe GitHub doesn't have any active tasks now, which is why it was rejecting every time

**Reply by CodeCraftman2** [reply]
What does it mean "every time” is it with any other person?

**Reply by CodeCraftman2** [reply]
What does it mean "every time” is it with any other person?

**Reply by pperlee** [reply]
I have the same question, after I finished the course, I create a new classroom and a new assignment, but still not work.

**Reply by pperlee** [reply]
![Screenshot 2024-04-22 at 00-21-31 GitHub Classroom](https://github.com/community/community/assets/57145799/d76bfda9-1091-4e72-97aa-a54e1c516cae)
maybe the reason comes from above (when i clicked "update" under the repo, the url went 404.)

**Reply by gurjeetsinghvirdee** [reply, marked as answer]
@CodeCraftman2 
**Every time** means that on each occasion the task is attempted, it is rejected consistently. It doesn't imply involvement of another person;

**Comment by gurjeetsinghvirdee** [comment]
Creating a new classroom doesn't work for this; we have to complete the task and update that link, but it was broken, @pperlee . Since the task started in September 2023, and the time has already passed, I think they closed the forum to accept tasks.

**Reply by ULTRONLORD** [reply]
We were unable to find a Classroom assignment for your account.
Same issue here  @github help out
